src/components/ValueSet/CodingField.tsx

```diff
@@ -81,13 +86,14 @@ export const CodingField = ({
                 }}
               />
             </FormControl>
+            <FormMessage>{error?.message}</FormMessage>
```

Comment:
see how `FormMessage` is used in other places in care... we don't need to de-construct `fieldState` and pass it. shadcn-ui's form components does these automatically!